noxfile.py

```diff
@@ -4,13 +4,18 @@
 # The following comment should be removed at some point in the future.
 # mypy: disallow-untyped-defs=False
 
-import io
+import glob
 import os
 import shutil
 import subprocess
+import sys
 
 import nox
 
+sys.path.append(".")
+from tools.automation import release  # isort:skip  # noqa
```

Comment:
Not sure how I feel about that :see_no_evil:

Reply:
Let's just agree that none of us are happy about it, but I'm glad that this works. :P